x-pack/platform/plugins/private/canvas/public/components/color_palette/color_palette.tsx

```diff
@@ -44,25 +44,25 @@ export const ColorPalette: FC<Props> = ({
   }
 
   colors = colors.filter((color) => {
-    return tinycolor(color).isValid();
+    return chroma.valid(color);
   });
 
   return (
     <div className="canvasColorPalette">
       <ItemGrid items={colors} itemsPerRow={colorsPerRow}>
         {(color) => {
-          const match = tinycolor.equals(color, value);
+          const match = chroma(color).name() === chroma(value).name();
```

Comment:
as previously stated, this doesn't match semi-transparent colors with a specific name/keyword

Reply:
solved by https://github.com/elastic/kibana/commit/666ce329ce93a8a616919c94170a20d339ae51e7